tests/acceptance/test_performance_trace_detail.py

```diff
@@ -194,5 +194,7 @@ def test_with_data(self, mock_now):
         with self.feature(FEATURE_NAMES):
             self.browser.get(self.path)
             self.browser.wait_until_not('[data-test-id="loading-indicator"]')
-            self.browser.elements('[data-test-id="transaction-row-title"]')[1].click()
+            row_title = self.browser.elements('[data-test-id="transaction-row-title"]')[1]
+            # HACK: Use JavaScript to execute click to avoid click intercepted issues
+            self.browser.driver.execute_script("arguments[0].click()", row_title)
```

Comment:
Inspired by stackoverflow and existing example

https://github.com/getsentry/sentry/blob/master/fixtures/page_objects/dashboard_detail.py#L44-L45